client/allocator/src/freeing_bump.rs

```diff
@@ -88,11 +88,9 @@ fn error(msg: &'static str) -> Error {
 /// A custom "trace" implementation that is only activated when `feature = std`.
 ///
 /// Uses `wasm-heap` as default target.
-macro_rules! trace {
-	( $( $args:expr ),+ ) => {
-		sp_std::if_std! {
-			log::trace!(target: "wasm-heap", $( $args ),+);
-		}
+macro_rules! log {
+	($level:tt, $( $args:expr ),+ ) => {
+		log::$level!(target: "wasm-heap", $( $args ),+);
```

Comment:
TBH, this macro looks (at the call sites) a bit weird. Why don't we just remove it altogether, and stick to a regular logging as found in the rest of codebase?

Reply:
this is a patter that I use a lot in pallets, but I will respect the fact that this is the client realm and most logs are not like this, so will revert.

Reply:
FWIW, my main preference for this is that I don't like repeating `target: "..."` all over the place.